### PR TITLE
vmm: Update comments related with enum Error struct in config.rs

### PR DESCRIPTION
**Pull request by optimistyzy**
Make the comments style be consistent

### CODE REVIEW

File: vmm/src/config.rs [hunk 1]
```diff
@@ -80,27 +80,27 @@ pub enum Error {
...
     /// Missing path from device,
     ParseDevicePathMissing,
-    /// Failed to parse vsock parameters
+    /// Failed parsing vsock parameters
     ParseVsock(OptionParserError),
-    /// Failed to parse restore parameters
+    /// Failed parseing restore parameters
     ParseRestore(OptionParserError),
-    /// Failed to parse SGX EPC parameters
+    /// Failed parseing SGX EPC parameters
```
Comment (MrXinWang) [inline vmm/src/config.rs:87]: Typo: parsing.

File: vmm/src/config.rs [hunk 2]
```diff
@@ -80,27 +80,27 @@ pub enum Error {
     ParseDevice(OptionParserError),
     /// Missing path from device,
     ParseDevicePathMissing,
-    /// Failed to parse vsock parameters
+    /// Failed parsing vsock parameters
     ParseVsock(OptionParserError),
-    /// Failed to parse restore parameters
+    /// Failed parseing restore parameters
```
Comment (MrXinWang) [inline vmm/src/config.rs:85]: Typo: parsing.

**Review by MrXinWang — CHANGES_REQUESTED** [review]
Typo

File: vmm/src/config.rs [hunk 3]
```diff
@@ -80,27 +80,27 @@ pub enum Error {
...
     ParseNuma(OptionParserError),
-    /// Failed to validate configuration
+    /// Failed validating configuration
     Validation(ValidationError),
     #[cfg(feature = "tdx")]
-    /// Failed to parse TDX config
+    /// Failed parsing TDX config
     ParseTdx(OptionParserError),
     #[cfg(feature = "tdx")]
     // No TDX firmware
```
Comment (MrXinWang) [inline vmm/src/config.rs:101]: Also, maybe unifying the "//" here to "///" is better?

File: vmm/src/config.rs (hunk 1 above)
Comment (optimistyzy) [reply vmm/src/config.rs:87]: ack

File: vmm/src/config.rs (hunk 2 above)
Comment (optimistyzy) [reply vmm/src/config.rs:85]: ack

File: vmm/src/config.rs (hunk 3 above)
Comment (optimistyzy) [reply vmm/src/config.rs:101]: ack

**Comment by optimistyzy** [comment]
> Please fix typos in this PR. Thank you!

sure

**Review by MrXinWang — APPROVED** [review]

**Review by sboeuf — APPROVED** [review]

**Pull request merged**